### PR TITLE
Added Italian translation and fixed some typos

### PR DESCRIPTION
**Pull request by DaveKeehl**
Since I'm a native Italian speaker, I wanted to contribute and add an Italian translation to the list of locales. I also found some typos in various text/markdown files in the repo.

### CODE REVIEW

**Comment by sy-b** [comment]
I am amazed by those typos. 😄

Thanks 😊

**Review by sy-b — APPROVED** [review]

**Review by FlorianPanten — APPROVED** [review]

File: Docs/Guide__Installing.md [hunk 1]
```diff
@@ -191,7 +191,7 @@ For now, you can try this
 
 ##### App from Play Store
 
-- Although app(s) which use this API exists but aren't they official.
+- Although app(s) which use this API exist but they aren't official.
```
Comment (Pustur) [inline Docs/Guide__Installing.md:194]: Shouldn't this be `Although app(s) which use this API exist, they aren't official.`?

File: Docs/Guide__Troubleshooting.md [hunk 2]
```diff
@@ -63,12 +63,12 @@ You can follow this guide:
...
 [Fix error NET::ERR CERT DATE INVALID - Your connection is not private - Windows 7 - 2021](https://youtu.be/JYZLxP2Z8G4)
 
-If you don't want to install the certificate from Google drive
+If you don't want to install the certificate from Google Drive
 
 - Here is the official link to the certificate [**x1.i.lencr.org**](http://x1.i.lencr.org/).
 - **You will have to close all the tabs** before downloading this certificate.
 
-**The thumb print of real certificate is `cabd2a79a1076a31f21d253635cb039d4329a5e8`**
+**The thumbprint of real certificate is `cabd2a79a1076a31f21d253635cb039d4329a5e8`**
```
Comment (Pustur) [inline Docs/Guide__Troubleshooting.md:71]: `fingerprint` maybe?

File: extension-description-store.txt [hunk 3]
```diff
@@ -1,15 +1,15 @@
 Return YouTube Dislike restores the ability to see dislikes on YouTube.
 
-If it doesn't work: open extensions tab (chrome://extensions/) disable this extension and enable it again. This is a bug in chromium that breaks the extension in some cases. This should fix most of the problems. Hopefully chromium team will fix this soon
+If it doesn't work: open extensions tab (chrome://extensions/) disable this extension and enable it again. This is a bug in chromium that breaks the extension in some cases. This should fix most of the problems. Hopefully the chromium team will fix this soon.
 
 Starting December 13th 2021, YouTube removed the ability to see dislikes from their API.
-This extension aims to restore power to users by using a combination of archived like and dislike data, as well as the likes and dislikes made by extension users to show the most accurate ratings.
+This extension aims to restore power to users by using a combination of archived likes and dislike data, as well as the likes and dislikes made by extension users to show the most accurate ratings.
```
Comment (Pustur) [inline extension-description-store.txt:6]: I think this was correct before

File: Extensions/combined/_locales/it/messages.json [hunk 4]
```diff
@@ -0,0 +1,50 @@
+{
+    "extensionName": {
+        "message": "Return YouTube Dislike"
+    },
+    "extensionDesc": {
+        "message": "Ritorna la possibilità di vedere i dislikes"
```
Comment (Pustur) [inline Extensions/combined/_locales/it/messages.json:6]: `Ritorna` could maybe be `Ripristina`

**Review by Pustur — COMMENTED** [review]
👍

File: Docs/Guide__Troubleshooting.md (hunk 2 above)
Comment (sy-b) [reply Docs/Guide__Troubleshooting.md:71]: Thumbprint is correct.

File: extension-description-store.txt (hunk 3 above)
Comment (sy-b) [reply extension-description-store.txt:6]: yes

File: Docs/Guide__Installing.md (hunk 1 above)
Comment (DavideViolante) [inline Docs/Guide__Installing.md:194]:
```suggestion
- Although app(s) which use this API exist, they aren't official.

```

File: Extensions/combined/_locales/it/messages.json (hunk 4 above)
Comment (DavideViolante) [inline Extensions/combined/_locales/it/messages.json:6]:
```suggestion
        "message": "Ripristina la possibilità di vedere i dislikes"
```

File: extension-description-store.txt (hunk 3 above)
Comment (DavideViolante) [inline extension-description-store.txt:6]:
```suggestion
This extension aims to restore power to users by using a combination of archived like and dislike data, as well as the likes and dislikes made by extension users to show the most accurate ratings.
```

**Comment by DaveKeehl** [comment]
Just committed your suggestions and corrections to this PR 👍🏻

**Review by Pustur — APPROVED** [review]

**Review by DavideViolante — APPROVED** [review]

**Review by sy-b — APPROVED** [review]

**Pull request merged**